app/presenters/contacts_finder_presenter.rb

```diff
@@ -1,105 +0,0 @@
-class ContactsFinderPresenter
```

Comment:
Will removing this file negatively affect the contacts index page? I guess not, 'cause you've removed it and checked it all still works, but just for peace of mind.